scripts/split-e2e-tests.ts

```diff
@@ -129,7 +171,8 @@ export type CircleCIConfig = {
 };
 
 function getTestFiles(dir: string, pattern = 'src/**/*.test.ts'): string[] {
-  return sortTestsBasedOnTime(glob.sync(pattern, { cwd: dir })).reverse();
+  // Todo: add reverse to run longest tests first
+  return sortTestsBasedOnTime(glob.sync(pattern, { cwd: dir })); // .reverse();
```

Comment:
Is this Todo addressable? Any reason not to go ahead and add `.reverse()`?

Reply:
I'm not sure why that todo is there. Why wouldn't we always run the shortest tests first? I left that line/todo there from before, I did not add it and it's ~a year old on master.